internal/gatewayapi/translator.go

```diff
@@ -27,6 +27,12 @@ const (
 	// OwningGatewayNamespaceLabel is the owner reference label used for managed infra.
 	// The value should be the namespace of the Gateway used to CRUD the Service.
 	OwningGatewayNamespaceLabel = "gateway.envoyproxy.io/owning-gateway-namespace"
+
+	// minEphemeralPort is the first port in the ephemeral port range.
+	minEphemeralPort = 1024
+	// privilegedPortShift is the constant added to the privileged port
+	// to convert it into an ephemeral port.
```

Comment:
> added to the privileged port

I don't see `privileged port` defined anywhere so it's difficult to understand this comment.

Reply:
https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Well-known_ports are also sometimes referred to as privileged ports because the user than can bind to such ports needs to have more privileges. I'll update this variable to `WellKnownPortShift` to keep it in line with the RFC https://datatracker.ietf.org/doc/html/rfc1340